usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -555,8 +555,32 @@
         LPAD(CAST(CAST(REGEXP_EXTRACT(COALESCE(transaction_fpds.legal_entity_congressional, transaction_fabs.legal_entity_congressional), '^[A-Z]*(\\d+)(?:\\.\\d+)?$', 1) AS SHORT) AS STRING), 2, '0')
             AS recipient_location_congressional_code,
         RL_DISTRICT_POPULATION.latest_population AS recipient_location_congressional_population,
-        -- TODO: include recipient_location_congressional_code_current derivation
-        'TEST CUR REC CONGR TS' AS recipient_location_congressional_code_current,
+        (CASE
+            WHEN (
+                COALESCE(transaction_fpds.legal_entity_country_code, transaction_fabs.legal_entity_country_code) <> 'USA'
+            ) THEN NULL
+            WHEN (
```

Comment:
I think you can apply the same idea, detailed in my previous comments, here as well.